mne/preprocessing/tests/test_ica.py

```diff
@@ -796,8 +849,7 @@ def test_n_components_and_max_pca_components_none():
 
     tempdir = _TempDir()
     output_fname = op.join(tempdir, 'test_ica-ica.fif')
-
-    ica = ICA(max_pca_components=max_pca_components,
+    ica = ICA(max_pca_components=max_pca_components, method=method,
               n_components=n_components, random_state=random_state)
     with warnings.catch_warnings(record=True):  # convergence
         ica.fit(epochs)
```

Comment:
Did these changes add much time to the testing compared to `master`? A few sec is okay (for this type of functionality), a minute is not (unless you've fully optimized it for speed already and there is no way to make it better without compromising the tests).

Reply:
By "these changes", do you mean testing also for picard? If yes, then it roughly doubles the testing time. 

I tried to speed up the tests a little bit here and there, but I don't think it's safe not to test picard in each setting?

Reply:
Yes I mean all the `tests` changes -- what is the rough overall change, then (10 sec, a minute)?

Reply:
On my machine, the updated full test takes ~ 30 sec to run (so it adds ~ 15 sec).

Also, there is a slow test which takes about half of the time.

Reply:
Okay thanks. That is a lot but I think we can live with it for ICA. Someday maybe we can speed it up with fewer channels, data points, etc.